src/native-common/View.tsx

```diff
@@ -69,29 +69,61 @@ export class View extends ViewBase<Types.ViewProps, {}> {
                 this._internalProps.pointerEvents = 'box-none';
             }
         }
+
+        if (RN.Platform.OS === 'windows') {
+            this._processDragAndDropProps();
+        }
+    }
+
+    private _processDragAndDropProps() {
+        for (const name of ['onDragEnter', 'onDragOver', 'onDrop', 'onDragLeave']) {
+            const handler = this._internalProps[name];
+
+            if (handler) {
+                this._internalProps.allowDrop = true; // TODO: let RN figure this out?
+
+                this._internalProps[name] = (e: React.SyntheticEvent) => {
+                    handler({
+                        dataTransfer: (e.nativeEvent as any).dataTransfer,
+
+                        stopPropagation() {
+                            if (e.stopPropagation) {
+                                e.stopPropagation();
+                            }
+                        },
+
+                        preventDefault() {
+                            if (e.preventDefault) {
+                                e.preventDefault();
+                            }
+                        },
+                    });
+                };
+            }
+        }
     }
 
-    private _isButton (viewProps: Types.ViewProps): boolean {
+    private _isButton(viewProps: Types.ViewProps): boolean {
         return !!(viewProps.onPress || viewProps.onLongPress);
     }
 
     render() {
         if (this.props.animateChildEnter || this.props.animateChildMove || this.props.animateChildLeave) {
             return (
                 <AnimateListEdits { ...this._internalProps }>
-                    { this.props.children }
+                    {this.props.children}
```

Comment:
Please add back the spaces. The coding convention we use throughout ReactXP includes spaces within curly bracket statements.